workflows/README.md

```diff
@@ -1 +1,58 @@
+# Getting started with Docker
+
+If you want to run these protocols interactively you will need to install Docker on your system. You can find more detailed instructions [here](https://docs.docker.com/install/). Alternatively we've developed snakefiles for automation that do not require installation. You can find those instructions below. 
+
+#### First, update your machine
```

Comment:
Since you're talking about both a host and a container that could (are) running Ubuntu, it's good to be unambiguous. Machine does not specify which one you're referring to, so use "host" or "container".

Reply:
Addressed in the commits below.